jenkins/jobs/lxc-build-android.yaml

```diff
@@ -0,0 +1,30 @@
+- job:
+    name: lxc-build-android
+    concurrent: true
+    description: Builds an android tarball from the provided source.
+    node: lxc-priv && amd64
+    project-type: freestyle
+
+    properties:
+      - build-discarder:
+          num-to-keep: 15
+
+    parameters:
+      - string:
+          name: sha1
+          description: Hash of the commit to build
+
+      - string:
+          name: branch
+          description: Source git branch
+
+    builders:
+      - shell: |-
+          cd /lxc
+          exec sudo /lxc/bin/build-android amd64 https://github.com/lxc/lxc master ${sha1} ${WORKSPACE}/
+
+    publishers:
+      - archive:
+          artifacts: lxc-android.tar.gz
+
+      - workspace-cleanup
```

Comment:
Our standard pattern for this is to do:
```
- workspace-cleanup:
  fail-build: false
```

Can you update them all?

Reply:
Done